include/FLAC/callback.h

```diff
@@ -168,12 +168,12 @@ typedef int (*FLAC__IOCallback_Close) (FLAC__IOHandle handle);
  *  a data source is not seekable by setting the \a seek field to \c NULL.
  */
 typedef struct {
-	FLAC__IOCallback_Read read;
-	FLAC__IOCallback_Write write;
-	FLAC__IOCallback_Seek seek;
-	FLAC__IOCallback_Tell tell;
-	FLAC__IOCallback_Eof eof;
-	FLAC__IOCallback_Close close;
+	FLAC__IOCallback_Read read;   ///< See FLAC__IOCallbacks
```

Comment:
Shall this not emit warnings with -ansi or -pedantic?

Reply:
Probably better to do that with /**< */ indeed. I copied it from C++ sources to C sources, didn't think of that